$OpenBSD: patch-t_1-basic_t,v 1.1.1.1 2009/02/02 22:59:23 simon Exp $

Fix use of barewords in tests, bug filed upstream at
http://rt.cpan.org/Public/Bug/Display.html?id=42943

--- t/1-basic.t.orig	Tue Sep 16 09:36:13 2008
+++ t/1-basic.t	Tue Sep 16 09:36:30 2008
@@ -58,7 +58,7 @@ print(($@ ? 'not ok' : 'ok'), ' ', ($test_count++), "\
 print(($@ ? 'not ok' : 'ok'), ' ', ($test_count++), "\n");
 
 eval {
-	$osd->set_colour(red);
+	$osd->set_colour('red');
 	$osd->string(0, 'Red test line 1');
 	delay();
 	$osd->string(1, 'Red test line 2');
@@ -82,7 +82,7 @@ eval {
 	$osd->set_shadow_offset(4);
 	$osd->string(1, 'Shadow colour test 1');
 	delay();
-	$osd->set_shadow_colour(green);
+	$osd->set_shadow_colour('green');
 	$osd->string(2, 'Shadow colour test 2');
 	delay();
 };
@@ -99,7 +99,7 @@ print (($@ ? 'not ok' : 'ok'), ' ', ($test_count++), "
 
 eval {
 	$osd->set_shadow_offset(0);
-	$osd->set_outline_colour(blue);
+	$osd->set_outline_colour('blue');
 	$osd->set_outline_offset(2);
 	$osd->string(1, 'Outline colour and offset test 1');
 	delay(1);
